src/dtx/dtx_common.c

```diff
@@ -91,13 +91,14 @@ dtx_free_dbca(struct dtx_batched_commit_args *dbca)
 }
 
 static void
-dtx_flush_committable(struct dss_module_info *dmi,
-		      struct dtx_batched_commit_args *dbca)
+dtx_flush_all(struct dss_module_info *dmi, struct dtx_batched_commit_args *dbca)
```

Comment:
"all" is some confused, what we flushed is the "committable" DTXs, not "all" DTXs that also contains local "prepared".

Reply:
What about 'dtx_flush_on_deregister()'?

Reply:
Why not "dtx_flush_committable" ?

Reply:
The whole dtx_batched_commit() is to flush committable records, so I think we'd like to use the function name to tell difference of this special flush. (it'll flush all committable records, then deregister)

Reply:
The term "flush" may be some confused, how about to call it as "drain"? "dtx_drain_committable"

Reply:
That looks good to me as well, I'll update once the patch needs be refreshed.